<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>get_list_lyrics</class>
 <widget class="QWidget" name="get_list_lyrics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>获取专辑/歌单歌词</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/resource.qrc">
    <normaloff>:/LDDC/img/icon/logo.png</normaloff>:/LDDC/img/icon/logo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/resource.qrc"/>
 </resources>
 <connections/>
</ui>
